types/types.go

```diff
@@ -153,6 +153,9 @@ type ImageDestination interface {
 	// uploaded to the image destination, true otherwise.
 	AcceptsForeignLayerURLs() bool
 
+	// AcceptsAnyOSSpecificImage returns true if the destination can store images for every operating system. False otherwise.
```

Comment:
Nit, please remove the blank line above, so that blank lines separate functional groups of methods. (The line above `AcceptsForeighLayerURLs` might also be dropped.)